django/db/models/sql/query.py

```diff
@@ -2035,16 +2035,14 @@ def __init__(self, connector, num_children, negated):
         self.num_children = num_children
         # Maps of table alias to how many times it is seen as required for
         # inner and/or outer joins.
```

Comment:
I'm not too familiar with this code -- should "outer" be removed from this comment?